docker-entrypoint.sh

```diff
@@ -17,8 +17,13 @@ DEFAULT_ARGS="--http-port 8081 --dir=${TELEGRAM_WORK_DIR} --temp-dir=${TELEGRAM_
 CUSTOM_ARGS=""
 
 if [ -n "$TELEGRAM_STAT" ]; then
-  CUSTOM_ARGS="${CUSTOM_ARGS} --http-stat-port=8082"
+  CUSTOM_ARGS="${CUSTOM_ARGS} --http-stat-port=$TELEGRAM_STAT"
```

Comment:
Avoid changing the port inside the container, with docker you can change the exposed port, the internal port shall always be 8082 for the http-stat-port (eg `docker run -p 8082:8888` will bind the stat port to 8888 from the external environment)

Reply:
But most people probably wanna use a reverse proxy to add auth anyway, so it wouldn't matter as long as they use the same at both places.

I'd say 
```bash
TELEGRAM_STAT=${TELEGRAM_STAT:-8082}
```
at the beginning of the script would solve that, setting a sane default but still allowing to change it.

@giuseppeM99 Why would you **_need_** it to be never changed by anyone in their own deployment?

Reply:
Is not that i need it to never be changed
I just don't want it changed _inside_ the docker container, instead i want docker to manage the port forwarding